tests/test_aggregates.py

```diff
@@ -35,6 +35,57 @@ def test_stable_sort(self):
             employers.extend(result['employer'] for result in results)
         assert len(set(employers)) == len(rows)
 
+    def test_by_state(self):
+        [
+            factories.ScheduleAByStateFactory(
+                committee_id='C0001',
+                cycle=2012,
+                total=50,
+                state='NY',
+                state_full='New York',
+                count=5,
+            ),
+            factories.ScheduleAByStateFactory(
+                committee_id='C0002',
+                cycle=2012,
+                total=150,
+                state='NY',
+                state_full='New York',
+                count=6,
+            ),
+            factories.ScheduleAByStateFactory(
+                committee_id='C0003',
+                cycle=2018,
+                total=100,
+                state='OT',
+                state_full='Other',
+            ),
+        ]
+        results = self._results(
+            api.url_for(
+                ScheduleAByStateView,
+                committee_id='C0001',
+                cycle=2012
+            )
+        )
+        self.assertEqual(len(results), 1)
```

Comment:
This test module appears to use both `assert` and `self.assertEqual` with the former being more common. This would be a good opportunity to use `assert` everywhere. E.g. line 36.